front/app/utils/configs/votingMethodConfig/messages.ts

```diff
@@ -78,11 +78,6 @@ export default defineMessages({
     id: 'app.utils.votingMethodUtils.midSentenceVotes',
     defaultMessage: 'votes',
   },
-  numberOfVotes: {
-    id: 'app.utils.votingMethodUtils.numberOfVotes',
-    defaultMessage:
-      '{numberOfVotes} {numberOfVotes, plural, =0 {{votesTerm}} one {{voteTerm}} other {{votesTerm}}}',
-  },
```

Comment:
Actually with different languages handling it in the code could become an issue. If there's a language where you'd put the number behind the word "vote", write right to left, etc. 

So if we'd need that again in the future and we need Crowdin as such, it's best to include the {numberOfVotes} part in your translation so translators can easily move it around. E.g.:

```
numberOfVotes: {
    id: 'app.utils.votingMethodUtils.numberOfVotes',
    defaultMessage:
      '{numberOfVotes, plural, =0 {{numberOfVotes} {votesTerm}} one {{numberOfVotes} {voteTerm}} other {{numberOfVotes} {votesTerm}}}',
  },
```

Reply:
Anyway, they can already make this custom change in Crowdin, even if you don't structure your string like this, and disregard the string that's been handed to them.